src/web_app/js/appcontroller.js

```diff
@@ -193,7 +193,14 @@ AppController.prototype.showRoomSelection_ = function() {
 AppController.prototype.finishCallSetup_ = function(roomId) {
   this.call_.start(roomId);
 
-  window.onbeforeunload = this.call_.hangup.bind(this.call_);
+  // Chrome apps can't use onbeforeunload.
+  if (!isChromeApp()) {
+    // Call hangup with async = false. Required to complete multiple
+    // clean up steps before page is closed.
+    window.onbeforeunload = function() {
+      this.call_.hangup(false);
+    }.bind(this);
```

Comment:
I had it that was, but it was causing odd side effects, possibly because onbeforeunload calls the function with an event object. For some reason it was causing one of those "Are you sure you want to leave this page?" pop ups when you closed the page.